pkg/pgmodel/migrate.go

```diff
@@ -390,14 +391,14 @@ func replaceSchemaNames(r io.ReadCloser) (string, error) {
 		return "", err
 	}
 	s := buf.String()
-	s = strings.ReplaceAll(s, "SCHEMA_CATALOG", catalogSchema)
-	s = strings.ReplaceAll(s, "SCHEMA_EXT", extSchema)
-	s = strings.ReplaceAll(s, "SCHEMA_PROM", promSchema)
-	s = strings.ReplaceAll(s, "SCHEMA_SERIES", seriesViewSchema)
-	s = strings.ReplaceAll(s, "SCHEMA_METRIC", metricViewSchema)
-	s = strings.ReplaceAll(s, "SCHEMA_DATA_SERIES", dataSeriesSchema)
-	s = strings.ReplaceAll(s, "SCHEMA_DATA", dataSchema)
-	s = strings.ReplaceAll(s, "SCHEMA_INFO", infoSchema)
+	s = strings.ReplaceAll(s, "SCHEMA_CATALOG", utils.CatalogSchema)
```

Comment:
These things don't look like they should be in `utils` package... 
How about we make a new package called `common`? Might make more sense IMO.